src/hooks/events.ts

```diff
@@ -76,12 +76,12 @@ export class MessageAddedEvent extends HookEvent {
 export class BeforeToolCallEvent extends HookEvent {
   readonly type = 'beforeToolCallEvent' as const
   readonly agent: AgentData
-  readonly toolUse: {
+  toolUse: {
```

Comment:
After looking a this, it seems like this could be confusing; the only thing we want folks to update there is `toolUse.input`.

Let's keep `toolUse` as read-only, and instead expose a new [writable] member on this that is `toolInput` that folks can change